src/Arkanis.Overlay.Host.Desktop/Workers/GameWindowTrackerOverlayEventRelayService.cs

```diff
@@ -0,0 +1,42 @@
+namespace Arkanis.Overlay.Host.Desktop.Workers;
+
+using Arkanis.Overlay.Domain.Abstractions.Services;
+using Microsoft.Extensions.Hosting;
+
+public class GameWindowTrackerOverlayEventRelayService(GameWindowTracker windowTracker, IOverlayEventControls overlayEventControls) : IHostedService
```

Comment:
Why not include this in the `GameWindowTracker` ? This way we have an extra layer that re-emits events when we could emit them directly.

Seems like we would do better to change the public events on the `GameWindowTracker` by implementing the required events.

Reply:
Also I just noticed that this requires a dependency initialization order, which adds to the complexity.